examples/msgpack/src/tests.rs

```diff
@@ -0,0 +1,48 @@
+extern crate rmp_serde;
```

Comment:
Can we not depend on the crate here? I liked that the msgpack binary was in the test case. Just the binary with a note of what the bytes are representing would be great.